edward/inferences.py

```diff
@@ -376,7 +368,7 @@ def __init__(self, model, data=Data(), params=None):
 
     def build_loss(self):
         x = self.data.sample(self.n_data)
-        z, _ = self.variational.sample()
+        z = self.variational.sample()
         self.loss = tf.squeeze(self.model.log_prob(x, z))
```

Comment:
I prefer `z` because it follows semantically from what is happening, i.e., we are sampling from `variational` to get a 1 x shape(z) output, where `z` is a point mass family.